### PR TITLE
Add Appium to apps

### PR DESCRIPTION
**Pull request by dpgraham**
(cc @jlipps)

### CODE REVIEW

File: apps/appium/appium.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name: Appium
```
Comment (jlipps) [inline apps/appium/appium.yml]: I'd call this "Appium Desktop" instead of just "Appium"

**Comment by zeke** [comment]
Thanks! I would suggest adding some keywords like hybrid, automation, webdriver, etc.

**Comment by dpgraham** [comment]
Just added some more keywords. Thanks.

**Comment by zeke** [comment]
👍

**Pull request merged**